lib/openshift/elasticsearch.rb

```diff
@@ -88,11 +88,11 @@ def node_selector(user: nil, quiet: true, cached: false)
     end
 
     def status(user: nil, quiet: true, cached: false)
-      return raw_resource(user: user, cached: cached, quiet: quiet).dig('status')
+      return raw_resource(user: user, cached: cached, quiet: quiet).dig('status') unless raw_resource.nil?
     end
 
     def cluster_status(user: nil, quiet: false, cached: false)
-      return status.dig('cluster')
+      return status(user: nil, quiet: false, cached: false).dig('cluster') unless status.nil?
```

Comment:
It should be `status(user: user, cached: cached, quiet: quiet)` so that parameters passed in `cluster_status` can be passed to `status`.